config/socket/socket.js

```diff
@@ -223,6 +223,23 @@ export default (io) => {
       }
     });
 
+    socket.on('drawCard', () => {
+      if (allGames[socket.gameID]) {
+        const thisGame = allGames[socket.gameID];
+        console.log('comparing', thisGame.players[0].socket.id, 'with', socket.id);
```

Comment:
Why do we have console log statements in source control?

Reply:
I don't really understand why it is added in the codebase but i think it helps to know what socket emit and track the game state and changes. personally, i was able to understand how socket emit and listen to event. basically, it really helped me to understand the codebase. but if it is necessary to remove all the console log it is okay by me but i feel some of my team member would  need it for now.